framework/src/main/java/org/tron/core/services/interfaceOnSolidity/RpcApiServiceOnSolidity.java

```diff
@@ -124,6 +125,10 @@ public void start() {
       // add lite fullnode query interceptor
       serverBuilder.intercept(liteFnQueryGrpcInterceptor);
 
+      if (parameter.isRpcReflectionServiceEnable()) {
```

Comment:
Should the parameter rpcReflectionServiceEnable control all the RpcApiServices? It seems to be not flexible to use only one parameter to control all .